internal/http/services/owncloud/ocdav/propfind/propfind.go

```diff
@@ -987,6 +987,10 @@ func mdToPropResponse(ctx context.Context, pf *XML, md *provider.ResourceInfo, p
 			)
 		}
 
+		if md.Name != "" {
```

Comment:
maybe add a fallback to file.Base(md.Path)? I don't think every storage driver returns name, yet.

Reply:
tracking as a dedicated issue in https://github.com/cs3org/reva/issues/3161

Reply:
This will result in Names like `"1284d238-aa92-42ce-bdc4-0b0000009157$534bb038-6f9d-4093-946f-133be61fa4e7!1887651e-4f0b-4d7b-8057-50a6f3945522"` on a PROPFIND for a single resource. That was the reason we needed this field anyways.

Imho if the driver doesn't support names the prop is empty